core-api/src/main/java/com/optimizely/ab/bucketing/UserProfileService.java

```diff
@@ -39,13 +45,13 @@
      * @return a Map representing the user's profile.
      * The returned {@code Map<String, Object>} of the user profile will have the following structure.
      * {
-     *     userIdKey : String userId,
-     *     decisionsKey : {@code Map<String, Map<String, String>>} decisions {
-     *          String experimentId : {@code Map<String, String>} decision {
-     *              variationIdKey : String variationId
-     *          }
-     *      }
-     *  }
+     * userIdKey : String userId,
```

Comment:
Any reason not to keep the indentation for the sample code here? I think the indentation is valid in this case

Reply:
done